packages/turbo/ios/RNVisitableViewController.swift

```diff
@@ -24,36 +24,79 @@ public protocol RNVisitableViewControllerDelegate {
   
 }
 
-class RNVisitableViewController: VisitableViewController {
-  
-  public var delegate: RNVisitableViewControllerDelegate?
-  
+class RNVisitableViewController: UIViewController, Visitable {
+  public var visitableViewControllerDelegate: RNVisitableViewControllerDelegate?
+
+  open weak var visitableDelegate: VisitableDelegate?
```

Comment:
I think we get this delegate from `Visitable` protocol automatically?

Reply:
No, we don't
![image](https://github.com/software-mansion-labs/react-native-turbo-demo/assets/44744462/bb049eaf-baf3-44ea-83e7-a76581c45cc4)